setup-scripts/setup-siad.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+set -e # exit on first error
+
+# Install Go 1.13.11
+wget -c https://dl.google.com/go/go1.13.11.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.13.11.linux-amd64.tar.gz
+rm go1.13.11.linux-amd64.tar.gz
+
+# add gopath to PATH and persist it in /etc/profile
+export PATH="${PATH}:/usr/local/go/bin:/home/user/go/bin"
+echo "export PATH=${PATH}" | sudo tee /etc/profile.d/go_path.sh
+
+# Sanity check that will pass if go was installed correctly.
+go version
+
+# Install Sia
+rm -rf ~/Sia
+git clone --depth 1 -b v1.4.8 https://gitlab.com/NebulousLabs/Sia.git ~/Sia
+make --directory ~/Sia
```

Comment:
Do we really need compile Sia here? Can't we use a binary bundle?

Reply:
I would be ok with that but I guess it's there in case we want to use an rc or beta release. By having everything prepared for recompilation, we can just `cd ~/Sia`, checkout the branch, build it and restart the services to be picked up.